fns.txt

```diff
@@ -0,0 +1,102 @@
+new_array
```

Comment:
fns.txt should be removed from the PR. It is a generated file by `v -debug`

Reply:
yes, please remove fns.txt